lib/howl/commands/app_commands.moon

```diff
@@ -131,25 +132,11 @@ command.register
         return false
 
 command.register
-  name: 'describe-input',
-  description: 'Shows information for an input'
-  input: 'input'
-  handler: (name) ->
-    def = inputs[name]
-    error "Unknown input '#{name}'" unless def
-    buffer = with ActionBuffer!
-      .title = "Input: #{name}"
-      \append "#{def.description}\n\n"
-
-    buffer.read_only = true
-    buffer.modified = false
-    editor = howl.app\add_buffer buffer
-
-command.register
   name: 'describe-signal',
   description: 'Describes a given signal'
-  input: 'signal'
-  handler: (name) ->
+  interactive: true
+  handler: ->
+    name = interact.select_signal!
     def = signal.all[name]
```

Comment:
Need to check whether a signal was choosen here, otherwise we get the error "Unknown signal 'nil'".